database_schema_and_api.md

```diff
@@ -0,0 +1,14 @@
+### Movies
+
+title
+poster
+
```

Comment:
should we add language here as well, we are supporting expats, and they don't wish to go for a movie with vietnamese dub? Right?